src/main/java/com/google/api/codegen/util/ProtoParser.java

```diff
@@ -96,8 +96,8 @@ public OperationData getLongRunningOperation(Method method) {
   @SuppressWarnings("unchecked")
   /* Returns if a field is required, according to the proto annotations. */
   private boolean isFieldRequired(Field field) {
-    List<FieldBehavior> fieldBehaviors =
-        (List<FieldBehavior>)
+    List<EnumValueDescriptor> fieldBehaviors =
+        (List<EnumValueDescriptor>)
             field.getOptionFields().get(AnnotationsProto.fieldBehavior.getDescriptor());
     return fieldBehaviors != null && fieldBehaviors.contains(REQUIRED.getValueDescriptor());
```

Comment:
The `fieldBehaviors.contains(REQUIRED.getValueDescriptor())` technically worked previously because `REQUIRED.getValueDescriptor()` is an `EnumValueDescriptor`, and `fieldBehaviors` was _actually_ a `List<EnumValueDescriptor>`, even though I had cast it to a `List<FieldBehavior>`.